### PR TITLE
Added Contributing Guidelines

### PR DESCRIPTION
**Pull request by gautamjajoo**
Added Contributing Guidelines file. Thanks!!
@AvaisP

### CODE REVIEW

**Comment by AvaisP** [comment]
Will review the PRs in 24 hours

**Comment by gautamjajoo** [comment]
No problem..

**Comment by AvaisP** [comment]
This seems like a copy paste of EvalAI.

**Comment by gautamjajoo** [comment]
@AvaisP Yes.. I took it from there. Will make some more changes!!

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -4,49 +4,55 @@ Thank you for your interest in contributing to Origami! Here are a few pointers
...
 
 ### Instructions to submit code
 
 Before you submit code, please talk to us via the issue tracker so we know you are working on it.
 
-Our central development branch is development. Coding is done on feature branches based off of development and merged into it once stable and reviewed. To submit code, follow these steps:
+Coding is done on feature branches and merged into it once stable and reviewed. To submit code, follow these steps:
 
-1. Create a new branch off of development. Select a descriptive branch name.
+## 1. Create a new branch off of development. Select a descriptive branch name.
```
Comment (AvaisP) [inline .github/CONTRIBUTING.md]: We branch out of master

**Comment by gautamjajoo** [comment]
@AvaisP Done the necessary changes

**Comment by gautamjajoo** [comment]
@deshraj Please review!

**Pull request merged**